tools/mascot/src/main/java/dk/alexandra/fresco/tools/mascot/online/OnlinePhase.java

```diff
@@ -0,0 +1,117 @@
+package dk.alexandra.fresco.tools.mascot.online;
+
+import dk.alexandra.fresco.framework.network.Network;
+import dk.alexandra.fresco.tools.mascot.BaseProtocol;
+import dk.alexandra.fresco.tools.mascot.MascotResourcePool;
+import dk.alexandra.fresco.tools.mascot.elgen.ElementGeneration;
+import dk.alexandra.fresco.tools.mascot.field.AuthenticatedElement;
+import dk.alexandra.fresco.tools.mascot.field.FieldElement;
+import dk.alexandra.fresco.tools.mascot.field.MultTriple;
+import dk.alexandra.fresco.tools.mascot.triple.TripleGeneration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.BiConsumer;
+
+public class OnlinePhase extends BaseProtocol {
```

Comment:
Please document this. It is not clear to me what it is intended for.

Reply:
Done.